iaso/api/accounts.py

```diff
@@ -18,12 +18,16 @@ class Meta:
 
     def update(self, account, validated_data):
         default_version = validated_data.pop("default_version", None)
+        user = self.context["request"].user
         if default_version is not None:
             sourceVersion = get_object_or_404(
                 SourceVersion,
                 id=default_version.id,
                 number=default_version.number,
             )
+            project = Project.objects.get(account=user.iaso_profile.account)  # type: ignore
```

Comment:
an account might be linked to multiple project so this query will probably fail in production.

it might be easier to filter directly in the get_object_or_404,  something like `SourceVersion.filter(data_source__project__account_user=user)` not sure of the exact syntax

Reply:
I couldn't make it work this way as projects is a many to many so I did it differently and adapted the tests to take into consideration this case